src/main/java/org/jabref/logic/importer/AuthorListParser.java

```diff
@@ -147,59 +140,37 @@ public AuthorList parse(@NonNull String listOfNames) {
         listOfNames = simpleNormalForm.authors;
         boolean andOthersPresent = simpleNormalForm.andOthersPresent;
 
-        // Handle case names in order lastname, firstname and separated by ","
-        // E.g., Ali Babar, M., Dingsøyr, T., Lago, P., van der Vliet, H.
-        final boolean authorsContainAND = listOfNames.toUpperCase(Locale.ENGLISH).contains(" AND ");
```

Comment:
This code was removed - therefore `org.jabref.logic.formatter.bibtexfields.NormalizeNamesFormatterTest` is failing. Please fix. 😅